cmd/kube-apiserver/app/server.go

```diff
@@ -176,7 +175,7 @@ func CreateServerChain(completedOptions completedServerRunOptions, stopCh <-chan
 	if err != nil {
 		return nil, err
 	}
-	apiExtensionsServer, err := createAPIExtensionsServer(apiExtensionsConfig, genericapiserver.NewEmptyDelegate())
+	apiExtensionsServer, err := createAPIExtensionsServer(apiExtensionsConfig, serviceResolver, webhook.CreateWebhookAuthResolverWrapper(kubeAPIServerConfig.GenericConfig.LoopbackClientConfig, proxyTransport), genericapiserver.NewEmptyDelegate())
```

Comment:
can we move `serviceResolver` and `authResolverWrapper` into the `apiExtensionsConfig`? Having them here and in the `New()` is a non-standard plumbing path reserved for very special cases.